cloud-consumption-api/src/main/java/com/sequenceiq/consumption/api/v1/consumption/endpoint/ConsumptionInternalEndpoint.java

```diff
@@ -57,4 +57,40 @@ ConsumptionExistenceResponse doesStorageConsumptionCollectionExist(@AccountId @Q
             @QueryParam("monitoredResourceCrn") String monitoredResourceCrn, @NotEmpty @QueryParam("storageLocation") String storageLocation,
             @ValidCrn(resource = { CrnResourceDescriptor.USER, CrnResourceDescriptor.MACHINE_USER })
             @QueryParam("initiatorUserCrn") @NotEmpty String initiatorUserCrn);
+
+    @POST
+    @Path("schedule/ebs")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = ConsumptionOpDescription.SCHEDULE_STORAGE, produces = MediaType.APPLICATION_JSON,
+            nickname = "scheduleEbsCollection")
+    void scheduleEbsConsumptionCollection(@AccountId @QueryParam("accountId") String accountId,
+            @Valid @NotNull StorageConsumptionRequest request,
+            @ValidCrn(resource = { CrnResourceDescriptor.USER, CrnResourceDescriptor.MACHINE_USER })
+            @QueryParam("initiatorUserCrn") @NotEmpty String initiatorUserCrn);
+
+    @DELETE
+    @Path("unschedule/ebs")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = ConsumptionOpDescription.UNSCHEDULE_STORAGE, produces = MediaType.APPLICATION_JSON,
+            nickname = "unscheduleEbsCollection")
+    void unscheduleEbsConsumptionCollection(@AccountId @QueryParam("accountId") String accountId,
+            @NotNull @ValidCrn(resource = {CrnResourceDescriptor.ENVIRONMENT, CrnResourceDescriptor.DATALAKE})
+            @QueryParam("monitoredResourceCrn") String monitoredResourceCrn,
+            @NotEmpty @QueryParam("ebsId") String ebsId,
+            @ValidCrn(resource = { CrnResourceDescriptor.USER, CrnResourceDescriptor.MACHINE_USER })
+            @QueryParam("initiatorUserCrn") @NotEmpty String initiatorUserCrn);
+
+    @GET
+    @Path("exists/ebs")
```

Comment:
exists unnecessary here, the GET always load the exists one or the list. 

Other question:
this is the queried resource? `monitoredResourceCrn` if yes, please add this to the URL like /ebs/{resource}.

Reply:
added it to  the url